lessons/PowerShell/managed-identity-powershell.md

```diff
@@ -0,0 +1,240 @@
+# Managed Identity (PowerShell)
+
+Watch the recording of this lesson [on YouTube]().
+
+## Goal 🎯
+
+The goal of this lesson is to understand how you can create and use a system assigned managed to call an Azure function in order to obtain a Microsoft Graph access token with the right permission scope. Microsoft Graph is THE API for all things Microsoft 365. 
+
+This lessons consists of the following exercises:
+
+|Nr|Exercise
+|-|-
+|0|[Prerequisites](#0-prerequisites)
+|1|[Create an Azure Functions](#1-create-an-azure-functions)
+|2|[Create Azure resources](#2-create-azure-resources)
+|3|[Create Managed Identity and assign permissions](#3-create-managed-identity-and-assign-permissions)
+|4|[Obtain an access token](#4-obtain-an-access-token)
+|5|[Homework](#5-homework)
+|6|[More info](#6-more-info)
+
+> 📝 **Tip** - If you're stuck at any point you can have a look at the [source code](../../src/{language}/{topic}) in this repository.
+
+---
+
+## 0. Prerequisites
+
+| Prerequisite | Exercise
+| - | -
+
+See [{language} prerequisites](../prerequisites/prerequisites-{language}.md) for more details.
+
+## 1. Create an Azure Functions
+
+before we will deploy our app to Azure, we will develop it locally in Visual Studio Code. This comes with some great advantages such as 
+
+* we don’t need to adjust to ever changing UI in Azure portal
+* we can benefit from source control
+* we can collaborate with others
+
+### Steps
+
+1. Install the Core Tools package with `npm install -g azure-functions-core-tools@3 --unsafe-perm true`
```

Comment:
You can skip 1 and 2 since they are also described in the prerequistes page.

Reply:
I suggest to use this formatting to keep the style the same as the other lessons:
https://github.com/marcduiker/azure-functions-university/blob/main/lessons/dotnetcore31/queue/queue-lesson-dotnet.md#steps-1

Reply:
ah, I see. I assumed that you'd do it the docs.microsoft.com way - but happy to adjust :-)

Reply:
✅